lib/csv_to_popolo.rb

```diff
@@ -339,30 +353,72 @@ def given?(key)
       @r.key?(key) && !@r[key].nil? && !@r[key].empty?
     end
 
+    def split_cell(key)
+      separator = MODEL[key][:multivalue_separator]
+      if separator
+        @r[key].split(separator)
+      else
+        [@r[key]]
+      end
+    end
+
+    def set_cell_from_values(key, values)
```

Comment:
Should the 'Person' class be explicitly modifying the row object? It seems like this sort of method should really live in the CSV class.